Process Process-11:
Traceback (most recent call last):
  File "C:\Users\notes\Anaconda3\envs\smawm\lib\multiprocessing\process.py", line 258, in _bootstrap
    self.run()
  File "C:\Users\notes\Anaconda3\envs\smawm\lib\multiprocessing\process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\notes\smawm\controller.py", line 182, in slave_routine
    r_queue.put((s_id, r_gen.rollout(params)))
  File "C:\Users\notes\smawm\controller.py", line 132, in rollout
    self.env.step(action)
  File "C:\Users\notes\Anaconda3\envs\smawm\lib\site-packages\pettingzoo\utils\wrappers\order_enforcing.py", line 68, in step
    super().step(action)
  File "C:\Users\notes\Anaconda3\envs\smawm\lib\site-packages\pettingzoo\utils\wrappers\base.py", line 68, in step
    self.env.step(action)
  File "C:\Users\notes\Anaconda3\envs\smawm\lib\site-packages\pettingzoo\utils\wrappers\assert_out_of_bounds.py", line 24, in step
    super().step(action)
  File "C:\Users\notes\Anaconda3\envs\smawm\lib\site-packages\pettingzoo\utils\wrappers\base.py", line 68, in step
    self.env.step(action)
  File "C:\Users\notes\Anaconda3\envs\smawm\lib\site-packages\pettingzoo\mpe\_mpe_utils\simple_env.py", line 211, in step
    self._execute_world_step()
  File "C:\Users\notes\Anaconda3\envs\smawm\lib\site-packages\pettingzoo\mpe\_mpe_utils\simple_env.py", line 144, in _execute_world_step
    self.world.step()
  File "C:\Users\notes\Anaconda3\envs\smawm\lib\site-packages\pettingzoo\mpe\_mpe_utils\core.py", line 130, in step
    self.integrate_state(p_force)
  File "C:\Users\notes\Anaconda3\envs\smawm\lib\site-packages\pettingzoo\mpe\_mpe_utils\core.py", line 187, in integrate_state
    entity.state.p_pos += entity.state.p_vel * self.dt
KeyboardInterrupt
